src/modules/bigquery.js

```diff
@@ -159,6 +160,40 @@ export default class BigQuery {
       .then((response) => response.data.rows);
   };
 
+  getAutocompleteForColumn = (
+    currText,
+    columnName,
+    dataset,
+    tableName,
+    token,
+    filterStatement,
+    joinStatement,
+  ) => {
+    const url = `https://bigquery.googleapis.com/bigquery/v2/projects/${dataset.dataProject}/queries`;
+    const filterOrEmpty = _.isEmpty(filterStatement)
+      ? `WHERE ${columnName} LIKE '%${currText}%'`
+      : `WHERE ${filterStatement} AND ${columnName} LIKE '%${currText}%'`;
```

Comment:
Will this kick off for every character entered in the filter panel?

Reply:
nope, i'm calling `getAutocompleteForColumnDebounced` in the component which will fire this off after the user stops typing for 500ms

Reply:
Nice, that's awesome.